src/osemgrep/cli_scan/Output.mli

```diff
@@ -1,5 +1,13 @@
 module Out = Semgrep_output_v1_j
 
+val preprocess_result :
+  Scan_CLI.conf -> Core_runner.result -> unit -> Out.cli_output
+(** [preprocess_result conf result] preprocesses the result of a scan
+  * according to the configuration [conf]. This handles
+  * nosemgrep, interolating messages, and more. It returns a function that
```

Comment:
I would remove the unit parameter and related comment.
The need to profile function should not require to change their signature. It just adds an extra
complexity that is not needed.

Reply:
yea I wasn't thinking, will fix it